src/commandsAndMenu.tsx

```diff
@@ -558,6 +562,34 @@ export function addCommands(
           shell.add(diffWidget, 'main');
           shell.activateById(diffWidget.id);
 
+          // Search for the tab
+          const dockPanel = (app.shell as any)._dockPanel as DockPanel;
+
+          let tabPosition = -1;
+          const tabBar = find(dockPanel.tabBars(), bar => {
+            tabPosition = bar.titles.indexOf(diffWidget.title);
+            console.log('2', tabPosition);
+            return tabPosition !== -1;
+          });
+
+          // Get the tab
+          setTimeout(() => {
+            const tab =
+              tabPosition >= 0
+                ? tabBar.contentNode.children[tabPosition]
+                : null;
+            const tabTitle = tab.querySelector<HTMLElement>(
+              '.lm-TabBar-tabLabel'
+            );
+            tabTitle.style.fontStyle = 'italic';
+            tabTitle.addEventListener('mouseup', () => {
+              tabTitle.style.fontStyle = 'normal';
+              if (PreviewMainAreaWidget.previewWidget === diffWidget) {
+                PreviewMainAreaWidget.previewWidget = null;
+              }
```

Comment:
Could you create a static method in `PreviewMainAreaWidget` like `pin(widget: Widget)` that will do that?